### PR TITLE
common.mk: misc updates

### PR DESCRIPTION
**Pull request by vchong**
Label phony targets as such
Remove $(EDK2_PATH)/Conf/.cache in edk2-clean-common
Update description for LINUX_CLEANER_COMMON_FLAGS

Signed-off-by: Victor Chong <victor.chong@linaro.org>

### CODE REVIEW

File: common.mk [hunk 1]
```diff
@@ -155,6 +159,7 @@ endif
 
 LINUX_COMMON_FLAGS ?= LOCALVERSION= CROSS_COMPILE=$(CROSS_COMPILE_NS_KERNEL)
 
+.PHONY: linuxx-common
```
Comment (igoropaniuk) [inline common.mk]: linux**x**-common

Comment (vchong) [inline common.mk]: oops.. thanks for spotting!

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by vchong** [comment]
Rebased, squashed and tag added. Thanks!

**Pull request merged**